docs/manage/parts-and-remotes.md

```diff
@@ -0,0 +1,122 @@
+
+Connections between robots are established using the best network path available.
+
+When you configure a remote or a sub-part, the main robot part can access all the components and services configured on the remote robot part as though they were entities of the main robot part.
+This is a one-way connection: The main robot part can access the resources of the remote robot part, but the remote robot cannot access the resources of the robot part remoting into it.
+
+![Example of a remote and a two part robot where the main (and only) part of robot 1 remotes into the main part of robot 2, and thus has access to all resources of robot 2.](../img/parts-and-remotes/remotes-diagram.png)
+
+## Configuration
+
+### Configure a sub-part
+
+You can make a multi-part robot by first configuring one part which is the "main" part, and then configuring one or more sub-parts.
+The main part will be able to access the resources of its sub-parts.
+Sub-parts will *not* have access to the resources of the main part.
+
+Use the parts drop-down menu in the top banner of your robot's page on the [Viam app](https://app.viam.com) to add a new sub-part:
+
+![The Viam app interface with the part drop-down open. A new part called "my-sub-part" is being created.](../img/parts-and-remotes/sub-part-config.png)
+
+<br>
+
+{{% alert title="Note" color="note" %}}
+
+When you create a sub-part, a connection is established between the main part and the sub-part without you needing to explicitly configure a remote.
+
+{{% /alert %}}
+
+### Configure a remote
+
+To establish a connection between a part of one robot and a part of a second robot, configure a `remote` on the first robot's part.
+
+1. Go to the Viam app robot page of the robot part to which you wish to establish the remote connection.
+   This is the robot part whose resources will be accessible to the other robot part.
+2. Click the **CODE SAMPLE** tab.
+3. On the **Language** toggle, select **Remotes**.
+4. Click **COPY JSON**.
+
+   ![The Viam app CODE SAMPLE tab with Remotes selected and a copyable JSON snippet with the name, address and secret of the robot part.](../img/parts-and-remotes/remote-address.png)
+
+5. Go to the Viam app robot page of the robot part from which you want to establish a remote connection.
+   This is the robot part that will be able to access the resources of the other robot part.
+6. Click the **CONFIG** tab, click the **REMOTES** sub-tab, and select **JSON** mode.
+
+   ![The Viam app CONFIG tab with the REMOTES sub-tab open and JSON mode selected.](../img/parts-and-remotes/remote-json-create.png)
+
+7. Click **Create Remote**.
+8. Paste the remote config you copied in step 4 into the empty field.
+9. Click **Save Config** in the bottom left of the screen.
+
+<!-- This is possibly wrong--should update with better understanding of auth key versus secret

```

Comment:
can you make a follow up ticket for that?

Reply:
Okay! https://viam.atlassian.net/browse/DOCS-606